requirements/base.txt

```diff
@@ -5,13 +5,10 @@
 #    make upgrade
 #
 appdirs==1.4.4            # via fs
-backports.os==0.1.1       # via fs
-django==1.11.29           # via edx-opaque-keys
+asgiref==3.2.10           # via django
+django==3.0.7             # via edx-opaque-keys
```

Comment:
Add in constraints

 Django<2.3

So we use the same version of edx-platform

Reply:
Added